deploy/terraform-vpn/variables.tf

```diff
@@ -0,0 +1,47 @@
+variable "name" {
+  type    = string
+  default = "cloud-burst"
+}
+
+variable "region" {
+  type    = string
+  default = "us-east-1"
+}
+
+variable "vpc_cidr" {
+  type    = string
+  default = "10.10.0.0/16"
+}
+
+variable "azs" {
+  type    = list(string)
+  default = ["us-east-1a", "us-east-1b", "us-east-1c"]
+}
+
+variable "local_cidrs" {
+  type    = list(string)
+  default = ["192.168.0.0/16", "172.16.0.0/12"]
+}
+
+variable "tunnel1_inside_cidr" {
+  type    = string
+  default = "169.254.10.20/30"
+}
+
+variable "tunnel2_inside_cidr" {
+  type    = string
+  default = "169.254.30.40/30"
+}
+
+variable "tunnel1_psk" {
```

Comment:
Might want to add something in the readme about how to generate these values.

Reply:
I updated the README, and added a link to the official docs. I also added descriptions to all variables in this file.